samza-sql/src/main/java/org/apache/samza/sql/translator/FilterTranslator.java

```diff
@@ -55,11 +55,13 @@ void translate(final LogicalFilter filter, final TranslatorContext context) {
 
     return inputStream.filter(message -> {
       Object[] result = new Object[1];
-      expr.execute(context.getExecutionContext(), context.getDataContext(), message.getFieldValues().toArray(), result);
+      expr.execute(context.getExecutionContext(), context.getDataContext(),
```

Comment:
Quick comment since you are modifying the XxxTranslator classes: can we add one unit test class per translator class in this module? I was working on merging the SamzaSql module w/ serialization PR and the biggest difficulties I had was that I need to create individual TestXxxTranslator class to do the whitebox testing. It would be great if you can add the unit test class per translator class here.

Reply:
As discussed offline, I have opened a ticket (SAMZA-1641) to track adding UTs for Translator classes.